Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/static/js/timeline.js

```diff
@@ -761,7 +795,75 @@ class TimelineFromEndpoint {
         if (!found)
             this.searchScale(null);
     };
+    highlightRange(baseDotRenderer, seriesResults, uuidRange) {
+        let leftX = null, rightX = null;
+        let lastRangeKey = null; // reset caches when range changes
+        const idxByUuid = new Map(seriesResults.map((r, i) => [r.uuid, i]));
+
+        return (data, context, x, y) => {
+            // toggle is off by default
+            if (!this.showRegressionHighlight) {
+                return baseDotRenderer(data, context, x, y);
+            }
+
+            const range = this.regRange;
+            if (!range) return baseDotRenderer(data, context, x, y);
```

Comment:
nit: you're using a lot of different conventions for one-line returns under `if` conditions. Here you have a one-liner, but on L806 you have it within `{}`, and on L751 you have separate lines without `{}`. I'd recommend choosing one for consistency.